Building MOLGENIS with empty local Maven repository fails

#### Reproduce
- Create a backup of your local Maven repository
- Empty your local Maven repository
- mvn clean install 
#### Expected
- Build succes
#### Observed
- Build failure
  `[ERROR] Failed to execute goal on project molgenis-das: Could not resolve dependencies for project org.molgenis:molgenis-das:jar:1.22.0-SNAPSHOT: Could not transfer artifact javax.jms:jms:jar:1.1 from/to java.net (https://maven-repository.dev.java.net/nonav/repository): Cannot access https://maven-repository.dev.java.net/nonav/repository with type legacy using the available connector factories: BasicRepositoryConnectorFactory: Cannot access https://maven-repository.dev.java.net/nonav/repository with type legacy using the available layout factories: Maven2RepositoryLayoutFactory: Unsupported repository layout legacy -> [Help 1]`

--- a/molgenis-app/pom.xml
+++ b/molgenis-app/pom.xml
@@ -65,9 +65,9 @@
 						<version>1.1.3.4.O</version>
 					</dependency>
 					<dependency>
-						<artifactId>log4j</artifactId>
-						<groupId>log4j</groupId>
-						<version>1.2.17</version>
+						<groupId>org.slf4j</groupId>
+						<artifactId>log4j-over-slf4j</artifactId>
+						<version>1.7.21</version>
 					</dependency>
 				</dependencies>
 			</plugin>

--- a/molgenis-app/pom.xml
+++ b/molgenis-app/pom.xml
@@ -52,6 +52,23 @@
 						<artifactId>mysql-connector-java</artifactId>
 						<version>${mysql.connector.java.version}</version>
 					</dependency>
+					<dependency>
+						<groupId>uk.ac.ebi.mydas</groupId>
+						<artifactId>mydas</artifactId>
+						<version>1.6.7</version>
+						<scope>system</scope>
+						<systemPath>${basedir}/../molgenis-das/src/main/resources/mydas-1.6.7.jar</systemPath>
+					</dependency>
+					<dependency>
+						<groupId>xpp3</groupId>
+						<artifactId>xpp3</artifactId>
+						<version>1.1.3.4.O</version>
+					</dependency>
+					<dependency>
+						<artifactId>log4j</artifactId>
+						<groupId>log4j</groupId>
+						<version>1.2.17</version>
+					</dependency>
 				</dependencies>
 			</plugin>
 		</plugins>

